src/components/RightSidebar/Participants/ParticipantsList/Participant/Participant.vue

```diff
@@ -360,23 +366,41 @@ export default {
 .participant-row {
 	display: flex;
 	align-items: center;
-	cursor: pointer;
+	cursor: default;
 	margin: 4px 0;
 	border-radius: 22px;
 	height: 56px;
 	padding: 0 4px;
 
+	&.isSearched {
+		cursor: pointer;
+	}
+
 	&__user-wrapper {
 		margin-top: -4px;
 		margin-left: 12px;
-		width: calc(100% - 96px);
+		width: calc(100% - 100px);
 		display: flex;
 		flex-direction: column;
+
+		&.has-call-icon {
+			/** reduce text width to have some distance from call icon */
+			padding-right: 5px;
+		}
+
+		&.has-call-icon {
```

Comment:
ERROR in 
src/components/RightSidebar/Participants/ParticipantsList/Participant/Participant.vue
 391:3  ✖  Unexpected duplicate selector ".participant-row__user-wrapper.has-call-icon", first used at line 386   no-duplicate-selectors